tensorboard/plugins/text_v2/text_v2_plugin_test.py

```diff
@@ -202,29 +174,11 @@ def test_reduce_and_jsonify(self):
         d3_expected = ([["foo", "bar"], ["zoink", "zod"]], (2, 2, 2), True)
         self.assertEqual(convert(d3), d3_expected)
 
-    @with_plugin(generate_testdata=False)
-    def testPluginIsActiveWhenNoRuns(self, plugin):
+    def testIsActiveReturnsFalse(self):
         """The plugin should be inactive when there are no runs."""
```

Comment:
The comment here is a bit stale since it's no longer the case that the plugin's inactive status depends on whether there are run or not.  I'd either remove it entirely, or replace it with `# `list_plugins` as called by TB core suffices` as commented in the code itself, or something else that indicates that it's intended to always return false.

Reply:
got it, good catch thanks, done